src/query/xasl_cache.c

```diff
@@ -1718,7 +1719,10 @@ xcache_invalidate_qcaches (THREAD_ENTRY * thread_p, const OID * oid)
 	      finished = true;
 	      break;
 	    }
-
+	  if (xcache_entry->list_ht_no < 0)
```

Comment:
Does it make sense to call xcache_Hashmap.erase() even if (xcache_entry->list_ht_no < 0)?

Reply:
xcache_entry->list_ht_no = -1, it means that the query cache is not available. And xcache_Hashmap.erase() is never called in this routine. I think of you confuse the routine 'xcache_invalidate_entries.

Reply:
I was confused.